### PR TITLE
Authorizes conditionally

### LINKED ISSUE #188: Limit Github Auth to only members of Maryville DevCenter or https://github.com/LaunchCodeEducation organizations

### PR DESCRIPTION
**Pull request by dane-johnson**
Closes #188 
- [x] Assures all requests come from a token granted to Codesplain.
- [x] Only allows logins from Maryville and Launchcode orgs
- [x] Revoke token if incorrect org

### CODE REVIEW

File: __mocks__/aws-sdk.js [hunk 1]
```diff
@@ -0,0 +1 @@
+module.exports = {}
```
Comment (solkaz) [inline __mocks__/aws-sdk.js]: Is Jest unable to mock this?

File: circle.yml [hunk 2]
```diff
@@ -7,7 +7,11 @@ dependencies:
     - yarn
 test:
   override:
-    - yarn test
+    - yarn test:
+        environment: #Don't expose our secrets while testing
+          CLIENT_ID: birdperson
+          CLIENT_SECRET: molting
```
Comment (solkaz) [inline circle.yml]: :bird: :man:

File: __mocks__/aws-sdk.js (hunk 1 above)
Comment (dane-johnson) [inline __mocks__/aws-sdk.js]: It isn't in the dependencies, so the mock fails.

File: circle.yml (hunk 2 above)
Comment (dane-johnson) [inline circle.yml]: 😆

File: __tests__/lambdas/authorizer.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
+const mockClientId = process.env.CLIENT_ID
+
+import { handler } from '../../awslambda/Authorize'
+
+const mockMethodArn = 'Teleport'
+
+const mockValidToken = 'ricksanchez'
+const mockInvalidToken = 'mortysmith'
+
+jest.mock('axios', () => {
```
Comment (solkaz) [inline __tests__/lambdas/authorizer.js]: Use `moxios` if you need to mock axios. Check the `app` actions test file for an example

**Pull request merged**